docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,900 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #186.        excl_zone = int(np.ceil(m / stumpy.config.STUMPY_EXCL_ZONE_DENOM))
</pre><p> </p><p>This line can be deleted as <code>excl_zone</code> is not being used anymore.  The <code>excl_zone</code> is now set to <code>m</code>  in the performant algorithm so as to remain faithful to the paper. </p><p> </p><p> </p><p><strong>Additional discussion: </strong></p><p>If we consider <code>T[:split_idx]</code> as the training part, and we want to make a decision about the new subsequence <code>T[split_idx : split_idx + m]</code> , what should we do? Should we compare it to the subsequences in <code>T[:split_idx]</code>   Or should we compare it to the subsequences in <code>T[:split_idx + m]</code> while ignoring the trivial zone? I think the former is the paper's approach. If we want to go with the latter, then we should bring back the <code>excl_zone</code> . </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>As mentioned above, the output from DAMP needs to be the same as the discord discovered via <code>stumpy.stump</code> .  Otherwise, we will waste a lot of time explaining it to users</p>

Reply:
I get the point. Later, I will bring back the `excl_zone` to make it behave like the discord discovery approach when using `stumpy.stump`